bootstrap_cfn/fab_tasks.py

```diff
@@ -322,35 +327,39 @@ def exit_maintenance(dry_run=False):
-    for elb in cfn_config.data['elb']:
-        if not apply_maintenance_criteria(elb):
-            continue
-        # obtain physical name from dict lookup, by converting elb name into safe name
-        # into logical name
-        phys_name = stack_elbs[mold_to_safe_elb_name(elb['name'])]
-
-        dns_name = [x.dns_name for x in full_load_balancers if x.name == phys_name]
-        if len(dns_name) == 1:
-            dns_name = dns_name[0]
-        else:
-            raise BootstrapCfnError(
-                "Lookup for elb with physical name \"{0}\" returned {1} load balancers, "
-                "while only exactly 1 was expected".format(phys_name, len(dns_name)))
-        zone_id = get_cached_zone_id(r53_conn, cached_zone_ids, elb['hosted_zone'])
-
-        # For record_value provide list of params as needed by function set_alias
-        # http://boto.readthedocs.org/en/latest/ref/route53.html#boto.route53.record.Record.set_alias
-        record_value = [
-            # alias_hosted_zone_id
-            R53.AWS_ELB_ZONE_ID[env.aws_region],
-            # alias_dns_name
-            dns_name,
-            # alias_evaluate_target_health (True/False)
-            False
-        ]
-        print green("Attempting to update: \"{0}\":{1}".format(elb, record_value))
-        r53_conn.update_dns_record(zone_name, zone_id, elb, 'A', record_value, is_alias=True, dry_run=dry_run)
-
+    try:
+        for elb in cfn_config.data['elb']:
+            if not apply_maintenance_criteria(elb):
+                continue
+            # obtain physical name from dict lookup, by converting elb name into safe name
+            # into logical name
+            phys_name = stack_elbs[mold_to_safe_elb_name(elb['name'])]
+
+            dns_name = [x.dns_name for x in full_load_balancers if x.name == phys_name]
+            if len(dns_name) == 1:
+                dns_name = dns_name[0]
+            else:
+                raise BootstrapCfnError(
+                    "Lookup for elb with physical name \"{0}\" returned {1} load balancers, "
+                    "while only exactly 1 was expected".format(phys_name, len(dns_name)))
+            zone_id = get_cached_zone_id(r53_conn, cached_zone_ids, elb['hosted_zone'])
+
+            # For record_value provide list of params as needed by function set_alias
+            # http://boto.readthedocs.org/en/latest/ref/route53.html#boto.route53.record.Record.set_alias
+            record_value = [
+                # alias_hosted_zone_id
+                R53.AWS_ELB_ZONE_ID[env.aws_region],
+                # alias_dns_name
+                dns_name,
+                # alias_evaluate_target_health (True/False)
+                False
+            ]
+            print green("Attempting to update: \"{0}\":{1}".format(elb['name'], record_value))
+            r53_conn.update_dns_record(get_zone_name(), zone_id, elb['name'], 'A', record_value, is_alias=True, dry_run=dry_run)
+    except KeyError:
```

Comment:
here I only added try/except to catch KeyError and boto Error. the rest of code remains the same